servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/README.md

```diff
@@ -187,3 +187,13 @@ terraform -chdir=./terraform destroy -auto-approve
 ```shell
 terraform -chdir=terraform destroy -auto-approve
 ```
+
+## Enhancement
+
```

Comment:
Let's add features like the way you update event hub binder readme, and the features should include: sync, resource provisioning, parititioning, session, error channels, message headers. And due to we don't show how to work in sync mode in this sample, thus we could only add text introduction without the code.

Reply:
Need we also udpate the event hub muti-binder sample?